client/src/redux/reducers/disclaimer.js

```diff
@@ -1,18 +1,19 @@
-import { SET_DISCLAIMER_MODAL } from '../actions/actions';
+import { DISCLAIMER_ANSWER } from '../actions/actions';
 
-const INITIAL_STATE = {
-    agreeToDisclaimer: false
-}
+const initialState = {
+    answer: null,
+};
```

Comment:
Was there any benefit of moving away from a boolean agree/disagree variable? Why not just have the initial state be false, and then only change to true when the user consents?

Reply:
Do we need to know if a user didn't click anything yet? What should happen when he disagrees?

Reply:
Good question. My thought process was basically that theoretically if a user logs in and clicks disagree, that'll do it's thing (decision pending on what exactly I guess, but say it includes logout) and if they log back in and the app remembers that user specifically clicked disagree, I think we would still want the popup to come up again. If it's just set to false I feel like it doesn't really differentiate from that case vs it's false because user has yet to click either option.

Reply:
So right now the logic is set like if the value is 'null' OR 'disagree', then popup again. I suppose that a bool would do just fine here, I guess I was thinking of a future case where we'd specifically want to know that the user clicked disagree.

Reply:
So I am a little confused, but I understand it like if you click disagree you would automatically logout? Then if you attempt to log in again you still need to be asked to agree, right? Is there a difference between when you log in for the first time and when you try to log in after disagreeing?

Reply:
As of now there isn't a difference. You're both right, I'll just change it to a bool and if we do come across a case where we need to know that user specifically clicked disagree it can be changed then.

Reply:
I don't think the click would determine when to redirect though.

The way I'm seeing it is you could keep it as a boolean value initialized to false. The user click disagree and an onclick (for example) callback redirects them to the WHO/CDC/Login/etc. This wouldn't necessarily update the state. It could still be false and the next time they open the app they'd still face this option.

I'm just not seeing a big differentiation from when they initially log into the app for the first time, or when they log back in after disagreeing once before. What do you think? Am I missing any context?

Reply:
Ah, just saw your message. Sounds good!

Reply:
updated it!